### DISCUSSION TITLE
Support for Google Maps Advanced Markers

### DISCUSSION THREAD

**Discussion by dominicabela** [react-native-maps/react-native-maps#4481] [Feature requests]
Google recently announced support for [Advanced Markers](https://cloud.google.com/blog/products/maps-platform/announcing-highly-customizable-higher-performance-markers) with better performance and much more customizability. Just wondering if there are plans to support this functionality within react-native-maps going forward.

**Comment by Tom-Sziler** [comment]
I came here to ask this exact thing. Just worked on a project where markers were very difficult to work with and the new advanced markers seem like they'll be a perfect fix for most of the issues I had, if not all.

**Comment by mHvNG** [comment]
Just a comment to show my interest in this feature.

**Comment by mateki0** [comment]
Hi, there's actually opened PR with this feature. If you have some time you can test it from branch or wait for merge :) 
https://github.com/react-native-maps/react-native-maps/pull/5039
